prime-router/src/main/kotlin/fhirengine/utils/FHIRBundleHelpers.kt

```diff
@@ -20,10 +22,16 @@ import java.util.stream.Stream
  * A collection of helper functions that modify an existing FHIR bundle.
  */
 object FHIRBundleHelpers {
+    const val observationUrl = "http://hl7.org/fhir/StructureDefinition/Observation"
```

Comment:
This needs to be a new extension and not just an observation. Create a new extension in prime-router/metadata/fhir_mapping/hl7/codesystem/ExtensionUrlMapping.yml

Reply:
conditionExtensionurl = "https://reportstream.cdc.gov/fhir/StructureDefinition/reportable-condition"